### PR TITLE
Updating folder structure

### PR DESCRIPTION
**Pull request by naiteeks**

### CODE REVIEW

File: ref-apps/rocket/readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Microsoft Rocket Video Analytics Platform
+
+Microsoft Rocket Video Analytics Platform is a highly extensible software stack that empowers everyone to build practical real-world live video analytics applications for object detection and counting/alerting with cutting edge machine learning algorithms.
```
Comment (anilmur) [inline ref-apps/rocket/readme.md:3]: build live video analytics applications such as object detection, object counting/alerting...

Comment (naiteeks) [reply ref-apps/rocket/readme.md]: modified.

File: utilities/video-analysis/notebooks/Yolov3/readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| [Yolo V3](yolov3-icpu-onnx/readme.md)             | HTTP      | iCPU | LVA module using YoloV3, a neural network for real-time object detection, that complies with the Open Neural Network Exchange (ONNX). Follow this sample if your solution requires an Intel CPU accelerated IoT Edge device with connection through HTTP. | |
+| [Tiny Yolo V3](tinyyolov3-icpu-onnx/readme.md)    | HTTP      | iCPU | LVA module using Tiny YoloV3, a lightweight variant of the Yolo V3 neural network model, that complies with the Open Neural Network Exchange (ONNX). Follow this sample if your solution requires an Intel CPU accelerated IoT Edge device with connection through HTTP. | |
+| [Tiny Yolo V3](http://aka.ms/)                    | gRPC      | iCPU | LVA module using Tiny YoloV3, a lightweight variant of the Yolo V3 neural network model, that complies with the Open Neural Network Exchange (ONNX). Follow this sample if your solution requires an Intel CPU accelerated IoT Edge device with connection through gRPC. gRPC is a remote procedure call that efficiently connects services in and across data centers with pluggable support for load balancing, tracing, health checking and authentication. | |
```
Comment (anilmur) [inline utilities/video-analysis/notebooks/Yolov3/readme.md:9]: What is an "Intel CPU accelerated IoT Edge device"? 
Will this not run on IoT Edge devices that use AMD as CPU?
Is it using special instruction sets specific to Intel?

Comment (anilmur) [reply utilities/video-analysis/notebooks/Yolov3/readme.md:9]: Also provide links to gRPC, ONNX, ...

File: utilities/video-analysis/notebooks/Yolov3/readme.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+| 3           | Windows     | 10 ([WSL](https://docs.microsoft.com/en-us/windows/wsl/about) on Ubuntu 18.04)  | 3.6.9             | Pip3        | Physical PC    | Azure VM running Ubuntu 18.04 |
+| 4           | Windows     | 10 (Git Bash)                                                                   | 3.8.3             | Pip3        | Physical PC    | Azure VM running Ubuntu 18.04 |
+
+## Terminology
+We refer to three different environments in this sample. Here are descriptions for each of them for future reference:
+
+<ol type="a">
+  <li>Azure Cloud Services: cloud-based services run on Azure datacenters.  </li>
+  <li>Development PC: the machine you are currently using to run this sample.</li>
+  <li>IoT Edge Device: another machine (be it a virtual machine or a computationally light powered mini PC) used to run LVA on the Edge. This IoT Edge device must be installed with a Debian-based Unix system with x64/AMD64 architecture. ARM processors are not supported yet.  </li>
```
Comment (anilmur) [inline utilities/video-analysis/notebooks/Yolov3/readme.md:27]: what is a "computationally light powered mini PC"?

File: utilities/video-analysis/notebooks/Yolov3/yolov3/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# YoloV3 folders - Under Construction
```
Comment (anilmur) [inline utilities/video-analysis/notebooks/Yolov3/yolov3/readme.md:1]: Path is weird:   notebooks/Yolov3/yolov3/readme.md

File: utilities/video-analysis/notebooks/Yolov3/readme.md (hunk 2 above)
Comment (naiteeks) [reply utilities/video-analysis/notebooks/Yolov3/readme.md]: @mustafakasap to help answer Anil's questions

File: utilities/video-analysis/notebooks/Yolov3/readme.md (hunk 3 above)
Comment (naiteeks) [reply utilities/video-analysis/notebooks/Yolov3/readme.md]: @mustafakasap  to help answer Anil's questions

File: utilities/video-analysis/notebooks/Yolov3/readme.md (hunk 2 above)
Comment (mustafakasap) [reply utilities/video-analysis/notebooks/Yolov3/readme.md]: we can change "Intel CPU accelerated IoT Edge device" into "IoT Edge device with X architecture". here X should come from LVA spec (x64/AMD64 ?)

File: utilities/video-analysis/notebooks/Yolov3/readme.md (hunk 3 above)
Comment (mustafakasap) [reply utilities/video-analysis/notebooks/Yolov3/readme.md]: Change this line from 
another machine (be it a virtual machine or a computationally light powered mini PC) used to run LVA on the Edge.

to

IoT edge device that is capable of running LVA and the AI extensibility module.

**Pull request closed without merging**